haiku-apps/eventual/eventual-1.0.recipe

```diff
@@ -0,0 +1,64 @@
+SUMMARY="A time management system"
+DESCRIPTION="Eventual is a personal time management system for Haiku. \
+Intended to be a full-featured organizer. "
+HOMEPAGE="https://sourceforge.net/projects/eventual/"
+COPYRIGHT="2011 Alex Hitech"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="846290b16e76a8a4ffbd955057306a18ac96d5e3"
+SOURCE_URI="https://github.com/HaikuArchives/Eventual/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="e0b9844cebb6ed2cf42f93e592fe2e4ce0b801457b31cbac7ff7f5b5d493f76e"
+SOURCE_FILENAME="Eventual-$srcGitRev.tar.gz"
+SOURCE_DIR="Eventual-$srcGitRev"
+PATCHES="eventual-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 ?x86_64"
+
+PROVIDES="
+	eventual = $portVersion
+	app:Eventual = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:make
+	cmd:sh
+	"
+
+BUILD()
+{
+	sh makelib.sh
+	cd src/Libraries
+	mkdir -p ../LibraryObjects
+	for LIBDIR in */ ; do
+		LIBDIR=${LIBDIR%?}
+		NAME=lib$LIBDIR.so
+		LIB=$LIBDIR/objects.x86-cc2-release/$NAME
+		cp $LIB ../EventEditor/$NAME
+		cp $LIB ../EventServer/$NAME
+		cp $LIB ../PreferencesPreflet/$NAME
+		cp $LIB ../LibraryObjects/$NAME
+	done
+	cd ../
+	cd EventEditor && make OBJ_DIR=objects && cd ../
+	cd EventServer && make OBJ_DIR=objects && cd ../
+	cd PreferencesPreflet && make OBJ_DIR=objects
+}
+
+INSTALL()
+{
+	mkdir -p $appsDir/Eventual
+	#mkdir -p $appsDir/Eventual/lib
+	cp -r 'src/EventEditor/objects/Eventual Editor' $appsDir/Eventual
+	cp -r 'src/EventServer/objects/Eventual Server' $appsDir/Eventual
+	cp -r 'src/PreferencesPreflet/objects/Eventual Preferences' $appsDir/Eventual
+	#cp -r src/LibraryObjects/ $appsDir/Eventual/lib
```

Comment:
OK. The problem comes from the `INSTALL()` funciton. It is not copying any lib. When the build fails you can:
```
cd /boot/home/haikuports/haiku-apps/eventual/work-1.0/packaging/
cd eventual
find . -type f -name "lib*.so*"
```
this should normally list all the libs, but it is not showing anything.


If you comment out the call to cp which copies the libraries, nobody else will install the shared libs :(

Reply:
Did I comment those lines? :(
I uncommented them, rebuilt the recipe, and now the libs show up when I type
`find . -type f -name "lib*.so*"`.

```
./apps/Eventual/lib/LibraryObjects/libActivity.so
./apps/Eventual/lib/LibraryObjects/lib[some-lib].so

...
```
But the build still gives me `POLICY ERROR`s (?)

Reply:
Oh wait... let me guess...
There shouldn't be `LibraryObjects` in the directory, right?